src/components/PageSection/index.tsx

```diff
@@ -60,6 +61,24 @@ const PageSection: React.FC<PageSectionProps> = ({
 }) => {
   const hasZeroMargin = !hasCurvedDivider || curvePosition === 'top'
 
+  const getPadding = () => {
+    // No curved divider
+    if (!hasCurvedDivider) {
+      return '48px 0'
+    }
+    // Bottom curved divider
+    // Less bottom padding, as the divider is present there
+    if (curvePosition === 'bottom') {
```

Comment:
I suppose it is this one? If yes do we need the top version? Don't see similar thing on top in Figma. Unless top version is just in case.
![Screen Shot 2021-05-31 at 9 57 00 AM](https://user-images.githubusercontent.com/81824236/120152785-96bcc000-c1f6-11eb-8586-862ed6d2e0a0.png)

Reply:
Yea the component is shared with the trading competition, which has a top version:

<img width="725" alt="Screenshot 2021-05-31 at 08 34 43" src="https://user-images.githubusercontent.com/79279477/120157571-147ace80-c1eb-11eb-9367-d5d6b80aa762.png">

Also soon to be used by the new home page - although no top version in those designs.